### ISSUE TITLE
FbView files feature

### ISSUE THREAD

**Issue by sergii-s** [full-build/full-build#100]
**Problem :**
All sln files are deleted after importing a repository to the fullbuild. This could obstruct user experience because for each imported repo/project they should create and open a view before starting to the work.
More so each user should do it as views are local and not sharable.
**Solution :**
Replace .sln files with .fbview files. Fbview file contains filters and flags used to create a view and can be commited. 
A file extention association should be configured in order to create and open .fbview file with a simple double click.
During the convert process the sln files should be replaced by .fbview files.
The command "fullbuild add view myprj ..... " should now create "myprj.fbview" file in the execution directory.

**Comment by pchalamet** [comment]
@sergii-s : could we schedule this for v3.1 ?

**Comment by sergii-s** [comment]
I have rebased the branch to the latest release v3 and have created a PR. Next step is to rework view commands : add view, open view, build view, publish view etc in order to search the view in the current folder only (instead of centreal repo). @pchalamet  if you ok with this I can continue

**Comment by pchalamet** [comment]
ok - I will get rid of develop. git flow is such a pita... I will merge v3.0 in master, then you will be able to merge this feature.
btw, could you please advise on https://github.com/full-build/full-build/issues/115 ?

**Comment by pchalamet** [comment]
ok struggled a little bit with git flow (with squash it's better).
you can merge on develop.

**Comment by sergii-s** [comment]
About #115 have no idea for now. Have to try it first. Only upgrade command will require administrative permissions ?

**Comment by sergii-s** [comment]
Squash commits are not so good idea for the gitflow I think. Loosing all the history

**Comment by pchalamet** [comment]
Technically, all code review happened and should be ok for squash merge. OK for (lost) history but is it important ? That's why release are gated now to track problems between versions instead of commits.

**Comment by pchalamet** [comment]
But the friction point with squash and git flow is that master and develop get out of sync.
That's just such a showstopper.
OK OK, I draw...

**Comment by pchalamet** [comment]
@sergii-s let me fix the mess between master and develop first. It's horrible. They should be in sync and they are just not.

**Comment by sergii-s** [comment]
In my opinion losing of the hisstory is important. It helps a lot while tracing some problem/regression or finding out why some logic was implemented in this way

**Comment by sergii-s** [comment]
ok I've cancelled the PR

**Comment by pchalamet** [comment]
completely pissed off. Wy the heck did I this squash merge... pfeuu.. 
sorry had to push --force to develop to be able to go forward.

**Comment by pchalamet** [comment]
you can go one, and rebase for the 4th time I guess

**Comment by pchalamet** [comment]
For searching views elsewhere, please open another issue and tell what you want to do to start discussing about it. tks.

**Comment by pchalamet** [comment]
Also could you try registering a file extension without administrator privilege ? Is it ok ?

**Comment by pchalamet** [comment]
Replaced by #122
